src/platform/device.gni

```diff
@@ -196,6 +196,15 @@ if (_chip_device_layer != "none" && chip_device_platform != "external") {
       "<platform/" + _chip_device_layer + "/SystemPlatformConfig.h>"
 }
 
+declare_args() {
+  # Enable jlink/segger_rtt support.
+  if (chip_device_platform == "qpg") {
+    chip_enable_segger_rtt = false
```

Comment:
how about:

```
# Enable jlink/segger_rtt support
chip_enable_segger_rtt = (chip_device_platform == "qpg")
```

Reply:
Yes, we can do that. We expect that more platforms will want to disable jlink/segger_rtt support, and then one-liner might become less readable. It’s your call.

Reply:
We have precedent for this:

https://github.com/project-chip/connectedhomeip/blob/master/src/platform/tests/BUILD.gn#L24
https://github.com/project-chip/connectedhomeip/blob/master/src/lib/core/BUILD.gn#L28

and I believe many other places